<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisualizationWindow</class>
 <widget class="QMainWindow" name="VisualizationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RIVET</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_main">
      <item row="1" column="0">
       <widget class="QGraphicsView" name="sliceView">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="sliceLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Line Selection Window</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="pdLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Persistence Diagram</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_controls">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>2</width>
            <height>2</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="2">
           <widget class="QDoubleSpinBox" name="angleDoubleSpinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>0</height>
             </size>
            </property>
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="maximum">
             <double>90.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="angleLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>&amp;Angle:</string>
            </property>
            <property name="buddy">
             <cstring>angleDoubleSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QDoubleSpinBox" name="offsetSpinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>0</height>
             </size>
            </property>
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>-1000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="offsetLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>&amp;Offset:</string>
            </property>
            <property name="buddy">
             <cstring>offsetSpinBox</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_1">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>2</width>
            <height>2</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>2</width>
            <height>2</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_xi">
          <item>
           <widget class="QLabel" name="BettiLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Bigraded Betti numbers</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer_8">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>1</width>
                <height>2</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="xi0CheckBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>0</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_9">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>1</width>
                <height>2</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="xi1CheckBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>1</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_10">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>1</width>
                <height>2</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="xi2CheckBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>2</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_11">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>1</width>
                <height>2</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>2</width>
            <height>2</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>2</width>
            <height>2</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_checkboxes">
          <item>
           <widget class="QCheckBox" name="normCoordCheckBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>&amp;Fit to window</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="barcodeCheckBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Show &amp;barcode</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" name="Bounds" columnstretch="0">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_bounds">
            <item>
             <layout class="QVBoxLayout" name="VerticalLayout_BottomCorner">
              <item>
               <layout class="QGridLayout" name="gridLayout_BottomCorner">
                <item row="1" column="0">
                 <widget class="QLabel" name="BottomCornerYLabel">
                  <property name="text">
                   <string>y min</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="BottomCornerXLabel">
                  <property name="text">
                   <string>x min</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QDoubleSpinBox" name="BottomCornerXSpinBox">
                  <property name="decimals">
                   <number>4</number>
                  </property>
                  <property name="singleStep">
                   <double>0.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QDoubleSpinBox" name="BottomCornerYSpinBox">
                  <property name="decimals">
                   <number>4</number>
                  </property>
                  <property name="singleStep">
                   <double>0.000000000000000</double>
                  </property>
                  <property name="value">
                   <double>0.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="VerticalLayout_TopCorner">
              <item>
               <layout class="QGridLayout" name="gridLayout_TopCorner">
                <item row="0" column="0">
                 <widget class="QLabel" name="TopCornerXLabel">
                  <property name="text">
                   <string>x max</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QDoubleSpinBox" name="TopCornerYSpinBox">
                  <property name="decimals">
                   <number>4</number>
                  </property>
                  <property name="singleStep">
                   <double>0.000000000000000</double>
                  </property>
                  <property name="value">
                   <double>5.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QDoubleSpinBox" name="TopCornerXSpinBox">
                  <property name="decimals">
                   <number>4</number>
                  </property>
                  <property name="singleStep">
                   <double>0.000000000000000</double>
                  </property>
                  <property name="value">
                   <double>5.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="TopCornerYLabel">
                  <property name="text">
                   <string>y max</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>2</width>
            <height>2</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="QGraphicsView" name="pdView"/>
      </item>
      <item row="2" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_for_text">
        <item>
         <widget class="QLabel" name="filenameLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Input file:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="homdimLabel">
          <property name="text">
           <string>Homology dimension:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1008</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain_Window">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionSave_line_selection_window_as_image"/>
    <addaction name="actionSave_persistence_diagram_as_image"/>
    <addaction name="separator"/>
    <addaction name="actionAutomatically_reset_line"/>
    <addaction name="separator"/>
    <addaction name="actionRestore_default_window"/>
    <addaction name="actionBetti_number_window"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionOnline_help"/>
    <addaction name="actionReport_bug"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionConfigure"/>
   </widget>
   <addaction name="menuMain_Window"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Load new data...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save computed data...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About RIVET...</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>Configure...</string>
   </property>
  </action>
  <action name="actionSave_persistence_diagram_as_image">
   <property name="text">
    <string>Export persistence diagram as image...</string>
   </property>
  </action>
  <action name="actionSave_line_selection_window_as_image">
   <property name="text">
    <string>Export line selection window as image...</string>
   </property>
  </action>
  <action name="actionOnline_help">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Online help</string>
   </property>
  </action>
  <action name="actionReport_bug">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Report bug...</string>
   </property>
  </action>
  <action name="actionRestore_default_window">
   <property name="text">
    <string>Restore default window</string>
   </property>
   <property name="toolTip">
    <string>Restore default window</string>
   </property>
  </action>
  <action name="actionAutomatically_reset_line">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Automatically reset line</string>
   </property>
   <property name="toolTip">
    <string>Move line if if is not visible in the window</string>
   </property>
  </action>
  <action name="actionBetti_number_window">
   <property name="text">
    <string>Betti number window</string>
   </property>
   <property name="toolTip">
    <string>The smallest window containing all Betti numbers</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>sliceView</tabstop>
  <tabstop>pdView</tabstop>
  <tabstop>angleDoubleSpinBox</tabstop>
  <tabstop>offsetSpinBox</tabstop>
  <tabstop>xi0CheckBox</tabstop>
  <tabstop>xi1CheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
